### PR TITLE
Exercises updates

### PR DESCRIPTION
**Pull request by lizroth**
*Issue #, if available:* n/a

*Description of changes:*

* Add reference/button to IAM CFN template in environment setup.
* Add Maven targets to Exercise 5 instructions.
* Some cleanup and wording suggestions to Ex5. (The general change for moving the example around / changing how the grants work is still pending.)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/exercises/5-multi_region.rst [hunk 1]
```diff
@@ -348,14 +354,15 @@ in us-west-2.
 
     .. group-tab:: Java
 
+
         We have built a simple bash script that sets the grant, thereby disabling the use of the CMK in us-east-2.
```
Comment (ksamreen) [inline docs/exercises/5-multi_region.rst]: should be CMK in us-west-2

File: docs/exercises/5-multi_region.rst [hunk 2]
```diff
@@ -348,14 +354,15 @@ in us-west-2.
...
         Run the script as below.
 
         Note, be sure to save the grant_id that outputs to the CLI. You will need this to revoke the grant.
```
Comment (ksamreen) [inline docs/exercises/5-multi_region.rst]: "Note, be sure to save the grant_id that outputs to the CLI. You will need this to revoke the grant.
Write"
They don't need to save this anymore the updated bash scripts write and pull the grant id from a txt file

File: docs/exercises/5-multi_region.rst (hunk 1 above)
Comment (lizroth) [reply docs/exercises/5-multi_region.rst]: Per offline discussion, fixing this in the code update pass

File: docs/exercises/5-multi_region.rst (hunk 2 above)
Comment (lizroth) [reply docs/exercises/5-multi_region.rst]: Per offline discussion, fixing this in the code update pass

**Comment by lizroth** [comment]
Got verbal approval from ksamreen since we're pushing the description updates to the code change PR, so going ahead and merging

**Pull request merged**